tap_hubspot/streams.py

```diff
@@ -93,3 +88,69 @@ def schema(self) -> dict:
         if self.cached_schema is None:
             self.cached_schema, self.properties = self.get_custom_schema()
         return self.cached_schema
+
+class PropertiesStream(HubspotStream):
+    """Define custom stream."""
+    schema_filepath = None
+    primary_keys = ["name"]
+    replication_key = "name"
```

Comment:
⚠️ Replication key should be something like `updatedAt` otherwise we will never update the records in BigQuery (assuming the name doesn't change)

Reply:
I agree, but some properties didn't have an updatedAt or createdAt value. IWould you think it would be better just to add an artificial one?

Reply:
Ah that's weird, I would have expected this value to always be there when I read their documentation 🤔 
Like why wouldn't there be at least a createdAt property?

Reply:
Maybe it's like that only for the Default Hubspot properties as opposed to the Potloc Custom Properties we created. If they are default, then they don't really have a creation date and I don't think we can edit them so an updatedAt column also doesn't make sense!